### ISSUE TITLE
A cloned proposal was not able to save and continue

### ISSUE THREAD

**Issue by mutambaraf** [UserOfficeProject/user-office-project-issue-tracker#1191]
During the last round a user was not able to save and continue on a cloned proposal because it had a previous answer that was not matching any of the new selectable answers this is shown below.
![image](https://github.com/user-attachments/assets/c66190e2-9e3d-46ee-8c1b-95f971745348)

### What is the problem and why is it a problem
Users who clone proposals with template select drop down selectable answers updated in the template.

**Comment by simonfernandes** [comment]
Farai and I briefly discussed that this seems to have occurred a few times this round, where the user has cloned a proposal and a question has changed. A possibility could be clearing the answer where the cloned question config differs to the existing question, forcing the user to select a fresh answer.

Something to be aware of @Cat-Lucifer

Other seemingly related issues:
https://github.com/UserOfficeProject/issue-tracker/issues/1181
https://github.com/UserOfficeProject/issue-tracker/issues/1159
https://github.com/UserOfficeProject/issue-tracker/issues/1192

**Comment by Cat-Lucifer** [comment]
@simonfernandes I was aware of this one yesterday - thanks for creating an issue for it.  I didn't change the selectable answers here - I'm especially careful about this on reportable Qs - so not sure what's happened.

**Comment by simonfernandes** [comment]
@Cat-Lucifer Farai will correct me if I'm wrong here but I think the problem was that "Technical development" had changed to "Technique development" (or vice versa). Was that not the case? @mutambaraf

**Comment by Cat-Lucifer** [comment]
@simonfernandes Yep, I get that, but I'm not sure why it was changed.  Anyhoo, it doesn't matter now.  As you say, would be good to prevent this type of issue going forward.

**Comment by mutambaraf** [comment]
This will need monitoring.

**Comment by mutambaraf** [comment]
No planned
